tests/testthat/parse_comments/mixed-in.R

```diff
@@ -0,0 +1,7 @@
+# A file with text
```

Comment:
Can you include something we can indent already, e.g. a function call or an arithmetic expression with inline comments?